exercises/kindergarten-garden/.meta/version

```diff
@@ -0,0 +1 @@
+1.1.0
```

Comment:
It looks like this exercise should be version `1.1.1` as noted [here](https://github.com/exercism/problem-specifications/blob/master/exercises/kindergarten-garden/canonical-data.json) please ensure that this PR adheres to the correct canonical data.